src/HW_models/NRF_EGU.c

```diff
@@ -4,10 +4,925 @@
+        EGU0_EVENTS_TRIGGERED_12,
+        EGU0_EVENTS_TRIGGERED_13,
+        EGU0_EVENTS_TRIGGERED_14,
+        EGU0_EVENTS_TRIGGERED_15,
+    },
+    {
+        EGU1_EVENTS_TRIGGERED_0,
+        EGU1_EVENTS_TRIGGERED_1,
+        EGU1_EVENTS_TRIGGERED_2,
+        EGU1_EVENTS_TRIGGERED_3,
+        EGU1_EVENTS_TRIGGERED_4,
+        EGU1_EVENTS_TRIGGERED_5,
+        EGU1_EVENTS_TRIGGERED_6,
+        EGU1_EVENTS_TRIGGERED_7,
+        EGU1_EVENTS_TRIGGERED_8,
+        EGU1_EVENTS_TRIGGERED_9,
+        EGU1_EVENTS_TRIGGERED_10,
+        EGU1_EVENTS_TRIGGERED_11,
+        EGU1_EVENTS_TRIGGERED_12,
+        EGU1_EVENTS_TRIGGERED_13,
+        EGU1_EVENTS_TRIGGERED_14,
+        EGU1_EVENTS_TRIGGERED_15,
+    },
+    {
+        EGU2_EVENTS_TRIGGERED_0,
+        EGU2_EVENTS_TRIGGERED_1,
+        EGU2_EVENTS_TRIGGERED_2,
+        EGU2_EVENTS_TRIGGERED_3,
+        EGU2_EVENTS_TRIGGERED_4,
+        EGU2_EVENTS_TRIGGERED_5,
+        EGU2_EVENTS_TRIGGERED_6,
+        EGU2_EVENTS_TRIGGERED_7,
+        EGU2_EVENTS_TRIGGERED_8,
+        EGU2_EVENTS_TRIGGERED_9,
+        EGU2_EVENTS_TRIGGERED_10,
+        EGU2_EVENTS_TRIGGERED_11,
+        EGU2_EVENTS_TRIGGERED_12,
+        EGU2_EVENTS_TRIGGERED_13,
+        EGU2_EVENTS_TRIGGERED_14,
+        EGU2_EVENTS_TRIGGERED_15,
+    },
+    {
+        EGU3_EVENTS_TRIGGERED_0,
+        EGU3_EVENTS_TRIGGERED_1,
+        EGU3_EVENTS_TRIGGERED_2,
+        EGU3_EVENTS_TRIGGERED_3,
+        EGU3_EVENTS_TRIGGERED_4,
+        EGU3_EVENTS_TRIGGERED_5,
+        EGU3_EVENTS_TRIGGERED_6,
+        EGU3_EVENTS_TRIGGERED_7,
+        EGU3_EVENTS_TRIGGERED_8,
+        EGU3_EVENTS_TRIGGERED_9,
+        EGU3_EVENTS_TRIGGERED_10,
+        EGU3_EVENTS_TRIGGERED_11,
+        EGU3_EVENTS_TRIGGERED_12,
+        EGU3_EVENTS_TRIGGERED_13,
+        EGU3_EVENTS_TRIGGERED_14,
+        EGU3_EVENTS_TRIGGERED_15,
+    },
+    {
+        EGU4_EVENTS_TRIGGERED_0,
+        EGU4_EVENTS_TRIGGERED_1,
+        EGU4_EVENTS_TRIGGERED_2,
+        EGU4_EVENTS_TRIGGERED_3,
+        EGU4_EVENTS_TRIGGERED_4,
+        EGU4_EVENTS_TRIGGERED_5,
+        EGU4_EVENTS_TRIGGERED_6,
+        EGU4_EVENTS_TRIGGERED_7,
+        EGU4_EVENTS_TRIGGERED_8,
+        EGU4_EVENTS_TRIGGERED_9,
+        EGU4_EVENTS_TRIGGERED_10,
+        EGU4_EVENTS_TRIGGERED_11,
+        EGU4_EVENTS_TRIGGERED_12,
+        EGU4_EVENTS_TRIGGERED_13,
+        EGU4_EVENTS_TRIGGERED_14,
+        EGU4_EVENTS_TRIGGERED_15,
+    },
+    {
+        EGU5_EVENTS_TRIGGERED_0,
+        EGU5_EVENTS_TRIGGERED_1,
+        EGU5_EVENTS_TRIGGERED_2,
+        EGU5_EVENTS_TRIGGERED_3,
+        EGU5_EVENTS_TRIGGERED_4,
+        EGU5_EVENTS_TRIGGERED_5,
+        EGU5_EVENTS_TRIGGERED_6,
+        EGU5_EVENTS_TRIGGERED_7,
+        EGU5_EVENTS_TRIGGERED_8,
+        EGU5_EVENTS_TRIGGERED_9,
+        EGU5_EVENTS_TRIGGERED_10,
+        EGU5_EVENTS_TRIGGERED_11,
+        EGU5_EVENTS_TRIGGERED_12,
+        EGU5_EVENTS_TRIGGERED_13,
+        EGU5_EVENTS_TRIGGERED_14,
+        EGU5_EVENTS_TRIGGERED_15,
+    }
+    };
```

Comment:
Minor: remove tab and add empty line after this array.

Reply:
done